forage-android/src/main/java/com/joinforage/forage/android/collect/VgsPinSubmitter.kt

```diff
@@ -34,13 +34,11 @@ internal class VgsPinSubmitter(
     ): ForageApiResponse<String> = suspendCoroutine { continuation ->
         val vgsCollect = buildVGSCollect(context)
         vgsCollect.bindView(foragePinEditText.getTextInputEditText())
-        val vgsInputField = foragePinEditText.getTextInputEditText()
 
         vgsCollect.addOnResponseListeners(object : VgsCollectResponseListener {
             override fun onResponse(response: VGSResponse?) {
-                // VGS wants us to call onDestroy, unlike Basis Theory
+                // Clear all information collected before by VGSCollect
                 vgsCollect.onDestroy()
-                vgsInputField.setText("")
```

Comment:
Have you tried this out in practice? I assume the `setText()` function was there because the PIN was still visible after submission.

Reply:
@dleis612 we call `forageEditText.clearText()` here https://github.com/teamforage/forage-android-sdk/pull/158/files#diff-66067141ba6138825bc49435c8ff29ee82f168779e92ce98ee46f10d7aafba8bR89

I kept the call to `onDestroy` inside that block -> where we have a reference to the `vgsCollect`

Reply:
Ok I see what you're saying. Just to confirm, we now call `clearText()` in a single place and the logic happens for both vaults when it does right?

Reply:
yep!